ES6/index.js

```diff
@@ -0,0 +1,74 @@
+import {
+  navBar, listLink, addLink, contactLink,
+} from '../modules/navBar.js';
+// import { Book } from "../modules/addRemoveBook.js";
+import { listBook, displayPage } from '../modules/bookListPage.js';
+import { anyRandomName, book, bookList } from '../modules/listOfBooks.js';
+import { luxon } from '../modules/luxon.js';
+
+const section = document.querySelector('.container');
+const mainForm = document.getElementById('book-form');
+const currentDate = document.createElement('div');
+currentDate.id = 'displayDateTime';
+
+// Imported from navBar.js
+// navBar;
```

Comment:
- [ ] Kindly remove all commented out and unused/ unwanted pieces of code from your code base. It makes it look cluttered and messy.

Reply:
Okay. thank you @Nemwel-Boniface. I will do that.